packages/react-core/src/components/Wizard/WizardToggle.tsx

```diff
@@ -83,20 +88,29 @@ export const WizardToggle: React.FunctionComponent<WizardToggleProps> = ({
           <CaretDownIcon aria-hidden="true" />
         </span>
       </button>
-      <div className={css(styles.wizardOuterWrap)}>
-        <div className={css(styles.wizardInnerWrap)}>
-          {nav(isNavOpen)}
-          <WizardBody
-            mainComponent={isInPage ? 'div' : 'main'}
-            aria-label={mainAriaLabel}
-            aria-labelledby={mainAriaLabelledBy}
-            hasNoBodyPadding={hasNoBodyPadding}
-          >
-            {activeStep.component}
-          </WizardBody>
+      <WizardDrawerWrapper
+        hasDrawer={hasDrawer && activeStep.drawerPanelContent}
```

Comment:
I think you can move this conditional out `WizardDrawerWrapper`. You should not be rendering it is these conditions are not met.